great_expectations/render/renderer/site_builder.py

```diff
@@ -573,8 +628,10 @@ def build(self):
             rendered_content = self.renderer_class.render(index_links_dict)
             viewable_content = self.view_class.render(rendered_content)
         except Exception as e:
-            logger.error("Exception occurred during data docs rendering: ", e, exc_info=True)
-            return None
+            logger.error('Exception occurred during data docs rendering: ', e, exc_info=True)
+            exception_traceback = traceback.format_exc()
+            exception_message = f'{type(e).__name__}: "{str(e)}".  Traceback: "{exception_traceback}".'
+            raise exceptions.GreatExpectationsError(exception_message)
```

Comment:
Again. I'm uncertain about this re-raise.

Reply:
Same motivation as above. Happy to discuss.